pkg/controllers/cassandra/cassandra.go

```diff
@@ -31,19 +35,21 @@ import (
 // It accepts a list of informers that are then used to monitor the state of the
 // target cluster.
 type CassandraController struct {
-	control             ControlInterface
-	cassLister          listersv1alpha1.CassandraClusterLister
-	cassListerSynced    cache.InformerSynced
-	serviceListerSynced cache.InformerSynced
-	queue               workqueue.RateLimitingInterface
-	recorder            record.EventRecorder
+	control                  ControlInterface
+	cassLister               listersv1alpha1.CassandraClusterLister
+	cassListerSynced         cache.InformerSynced
+	serviceListerSynced      cache.InformerSynced
+	statefulSetsListerSynced cache.InformerSynced
```

Comment:
(nit) statefulSets -> statefulSet (for consistency with other vars in this struct)

Reply:
Done